cmd/kubeadm/app/util/config/initconfiguration_test.go

```diff
@@ -34,16 +34,16 @@ import (
 )
 
 const (
-	masterV1alpha3YAML          = "testdata/conversion/master/v1alpha3.yaml"
-	masterV1alpha3YAMLNonLinux  = "testdata/conversion/master/v1alpha3_non_linux.yaml"
-	masterV1beta1YAML           = "testdata/conversion/master/v1beta1.yaml"
-	masterV1beta1YAMLNonLinux   = "testdata/conversion/master/v1beta1_non_linux.yaml"
-	masterInternalYAML          = "testdata/conversion/master/internal.yaml"
-	masterInternalYAMLNonLinux  = "testdata/conversion/master/internal_non_linux.yaml"
-	masterIncompleteYAML        = "testdata/defaulting/master/incomplete.yaml"
-	masterDefaultedYAML         = "testdata/defaulting/master/defaulted.yaml"
-	masterDefaultedYAMLNonLinux = "testdata/defaulting/master/defaulted_non_linux.yaml"
-	masterInvalidYAML           = "testdata/validation/invalid_mastercfg.yaml"
+	controlPlaneV1alpha3YAML          = "testdata/conversion/master/v1alpha3.yaml"
```

Comment:
we need to change `master` -> `controlplane` in the filenames and folder too if we are changing the constant names.

Reply:
yes, you were right ! 
I have just updated.

Reply:
awesome! thanks.
/ok-to-test